client/src/actions/query.js

```diff
@@ -6,6 +6,14 @@ export const EVENTS = [
 	'QUERY_END_DATE_CHANGE'
 ];
 
+// Setting the default query is not a Query event that should be listened to.
+export function setDefaultQuery( query ) {
```

Comment:
Another nitpick, now that there is an action that is not part of `EVENTS`, the name `EVENTS` is very generic and doesn't really say what it is for. I don't have a better suggestion, but if you can think of something else it might be good to change it.